star-dash/star-dash/GameEngine/Components/PowerUpComponent.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+class PowerUpComponent: Component {
+    let type: String
```

Comment:
Should we use an enum for the type of PowerUp?

Reply:
I was planning to resolve that issue when I implement 1-2 more power-ups in the next PR, is that okay? It will give me more context into what is probably a better design as there are other components that requires other types of information based on the type of power-up